datalad_registry/overview.py

```diff
@@ -56,8 +55,8 @@ def overview():  # No type hints due to mypy#7187.
                 row[col] = getattr(info, col)
 
             ts = info.info_ts
-            if ts:
-                row["last_update"] = time.strftime("%Y-%m-%dT%H:%M%z", time.gmtime(ts))
+            if ts is not None:
+                row["last_update"] = ts.strftime("%Y-%m-%dT%H:%M:%S%z")
             else:
                 row["last_update"] = None
```

Comment:
unrelated to this PR, since not changing this line but since not code covered.  I would assume that it ( `db.Column`?) is nullable (by default), thus this should not blow up

Reply:
Yes, [`nullable` (generally) defaults to `True`](https://docs.sqlalchemy.org/en/14/core/metadata.html#sqlalchemy.schema.Column.params.nullable).